test/jdk/java/text/Format/NumberFormat/DFSExponential.java

```diff
@@ -30,18 +30,18 @@
  * @run junit DFSExponential
  */
 
-import java.util.*;
-import java.text.*;
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+import java.util.Locale;
 
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.fail;
 
-public class DFSExponential
-{
-
+public class DFSExponential {
 
-   public void DFSExponenTest() throws Exception {
+    @Test
+    public void DFSExponenTest() throws Exception {
```

Comment:
Not your change, but the method name reads odd to me. `TestDFSExponential` may be better alined with the other test for serialization.

Reply:
Done, thanks for the review.